src/openassetio-core/CMakeLists.txt

```diff
@@ -36,6 +36,13 @@ install(TARGETS openassetio-core EXPORT ${PROJECT_NAME}_EXPORTED_TARGETS)
 
 # TODO(DF): Generate and install .pdb on Windows. See OCIO.
```

Comment:
When I looked at the CI output, it looked like this may already be done (I saw some reference in the command line to a .pdb).  So perhaps this TODO can simply be removed (should be evident from a local Windows build)?

Reply:
Installing pdbs with https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/244/commits/318b11dd272f399cf3745c1c40ca8ee603e13478

Reply:
Awesome! Obviously this isn't tested by CI but looks reasonable to me if you're happy it works OK.